CMakeLists.txt

```diff
@@ -56,11 +56,6 @@ endif()
 
 message(STATUS "UMD build type: ${CMAKE_BUILD_TYPE}")
 
-add_compile_definitions(
```

Comment:
Does this affect the debug build, i.e. would I still be able to build for debugging? I previously used this flag I think: `-DCMAKE_BUILD_TYPE=Debug`

Reply:
No it has no impact on debug build. This PR just removes an unused #define.

This looks like it was mistakenly copied from tt-metal at some point.